homeassistant/components/wallbox/sensor.py

```diff
@@ -42,4 +47,11 @@ def __init__(
     @property
     def native_value(self):
         """Return the state of the sensor."""
+        if (sensor_round := self.entity_description.precision) is not None:
+            try:
+                return round(
+                    self.coordinator.data[self.entity_description.key], sensor_round
+                )
+            except TypeError as ex:
+                raise IntegrationError from ex
```

Comment:
If we expect this to happen, ie it's more than a developer error, log an error instead and return None.

Reply:
OK, wil change this in new commit